es6-operators/main.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+
+v1 && console.log('v1 is truthy');
+v2 && console.log('v2 is truthy');
+
+/* TODO:
+ * Demonstrate how to assign a "default value" to a variable.
+ * In one expression, assign `'default-value'` to the variable `config1` if v1 is falsy.
+ * In one expression, assign `'default-value'` to the variable `config2` if v2 is falsy.
+ * Log both `config1` and `config2`.
+ * Expected: `config1 = { value: 'Something' }`, `config2 = 'default-value'`.
+ */
+
+const config1 = v1 || v1;
```

Comment:
`v1` shouldn't be on both sides. You should have a string of a default value on the other side